### PR TITLE
add .travis.yml file

### PR DESCRIPTION
**Pull request by ryan-williams**
See #10

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+  - "2.6"
```
Comment (danvk) [inline .travis.yml]: I don't particularly care about Python 2.6, otherwise LG.

**Comment by danvk** [comment]
I added you as a collaborator, feel free to merge at will!

**Comment by ryan-williams** [comment]
do you need to tell travis to run on this repo or will it just happen?

**Comment by danvk** [comment]
I flipped the switch over at travis-ci.org. I think the repo owner has to do it. The next push to any branch should trigger a build.

**Pull request merged**